modules/virt-sno-differences.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * virt/about-virt.adoc
+
+:_content-type: CONCEPT
+[id="virt-sno-differences_{context}"]
+= {sno-caps} {product-title} differences
```

Comment:
This now repeats as "Single-node OpenShift OpenShift Container Platform differences". I think you should remove the {product-title} now

Reply:
Done, thanks.